cvat-ui/src/components/create-task-page/create-task-content.tsx

```diff
@@ -442,6 +452,21 @@ class CreateTaskContent extends React.PureComponent<Props & RouteComponentProps,
             return;
         }
 
+        try {
+            this.validateQualityParams(quality);
```

Comment:
Why not return true/false like in other validation methods here?
Do we need to pass quality as a param if the method is reading from the class instance state anyway? And again it was not implemented the same way as in other validators.

Reply:
The method only checks one problem. So, the error message is anyway clear

Reply:
>  I'd like to see some automatic switching to random frame ordering if honeypots are selected in the quality section

Sounds like something easy to implement